.github/workflows/unit-tests.yml

```diff
@@ -40,8 +40,53 @@ jobs:
         mongo-version:
           - "4.4"
           - "7.0"
-    # We expect Django 4.0 to fail, so don't stop when it fails.
-    continue-on-error: ${{ matrix.django-version == '4.0' }}
+        include:
```

Comment:
I'm a little worried that this will cause new shards to not get added to the latest suite, should we use exclude syntax instead for the older python versions?